### PR TITLE
Fixed template text for teams page goal section

### PR DESCRIPTION
**Pull request by shadab**

### CODE REVIEW

File: src/views/Team.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
           <p
             class="google-font mt-3 mb-0"
             style="font-size:110%"
-          >Our goal is to organize space to connect the best industry experts with Indian audience to boost development of IT. And Our Core Team is:</p>
+          >Our goal is to build a community of developers who are interested in development technologies including but not lmited to google products. Our Core Team is:</p>
```
Comment (Omnighost547) [inline src/views/Team.vue]:
```suggestion
          >Our goal is to build a community of developers who are interested in development technologies including but not limited to Google products. Our Core Team is:</p>
```

Comment (tombolide) [inline src/views/Team.vue]:
```suggestion
          >Our goal is to build a community of developers who are interested in development technologies including but not limited to Google products. Our Core Team is:</p>
```

**Review by tombolide — CHANGES_REQUESTED** [review]
Formatting - see suggestion.

**Review by rossanodan — APPROVED** [review]
Good one guys! 👍

**Pull request merged**